index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
```

Comment:
Great job, kindly put an indentation of two spaces in your document, Indentation makes your works look perfect and is also good practice.